sqs2.tf

```
@@ -0,0 +1,18 @@
+locals {
+  sqs2_name = length(var.sqs2_name_override) > 0 ? var.sqs2_name_override : join("-", [var.name, "1"])
```

Shouldn't it be like so (`2` at the very end)?
```suggestion
  sqs2_name = length(var.sqs2_name_override) > 0 ? var.sqs2_name_override : join("-", [var.name, "2"])
```